### PR TITLE
updated tags

### PR DESCRIPTION
**Pull request by joeyk710**
Changed "starbin" to "pastebin", about me, third party and add to server tags

### CODE REVIEW

File: tags/tags.toml [hunk 1]
```diff
@@ -888,19 +890,21 @@ client.guilds.cache.reduce((acc, guild) => acc + guild.memberCount, 0)
...
-- Set authorization method to "In-app Authorization"
-- Check scopes `bot` and `applications.commands`
+**"Add App" button**
+- [Developer Portal](https://discord.com/developers/applications) > Your app > Installation
+- Choose "User Install" or "Guild Install"
+- Choose "Discord Provided Link" under "Install Link"
+- Check scopes `bot` and choose permissions that your app *needs* under "Default Install Settings"
 
-Image: https://i.imgur.com/taMHw7o.png
+Image: https://imgur.com/14pMHvK.png
```
Comment (Jiralite) [inline tags/tags.toml:907]:
```suggestion
Image: https://i.imgur.com/14pMHvK.png
```

File: tags/tags.toml [hunk 2]
```diff
@@ -11,7 +11,7 @@ if (<message>.author.bot) return;
 [bot-about-me]
 keywords = ["bam"]
 content = """
-To add text to the `About Me` section of your bot's profile, head [here](https://discord.com/developers/applications), click on your bot's application, and fill in the `Description`. The description supports full markdown as well (bold, underline, etc.).
+To give your bot an "About Me", head [here](https://discord.com/developers/applications), click on your application, and fill in the `Description` next to "General Information". The description supports full markdown as well (bold, underline, etc.).
```
Comment (Jiralite) [inline tags/tags.toml:14]: Maybe?

```suggestion
To give your application an "About Me", head [here](https://discord.com/developers/applications), click on your application, and fill in the `Description` next to "General Information". The description supports full markdown as well (bold, underline, etc.).
```

File: tags/tags.toml (hunk 1 above)
Comment (joeyk710) [reply tags/tags.toml:907]: gotcha, sorry about that. ill change it

File: tags/tags.toml (hunk 2 above)
Comment (joeyk710) [reply tags/tags.toml:14]: yeah that sounds better. ill change this as well

File: tags/tags.toml [hunk 3]
```diff
@@ -11,7 +11,7 @@ if (<message>.author.bot) return;
 [bot-about-me]
 keywords = ["bam"]
 content = """
-To add text to the `About Me` section of your bot's profile, head [here](https://discord.com/developers/applications), click on your bot's application, and fill in the `Description`. The description supports full markdown as well (bold, underline, etc.).
+To give your application an "About Me", head [here](https://discord.com/developers/applications), click on your application, and fill in the `Description` next to "General Information". The description supports full markdown as well (bold, underline, etc.).
```
Comment (almostSouji) [inline tags/tags.toml:14]: inconsistent quoting, decide on one

File: tags/tags.toml [hunk 4]
```diff
@@ -713,7 +713,9 @@ keywords = [
 ]
 content = """
 **We do not provide any help with third party libraries**
-Buttons and Modals are supported natively.
+discord.js supports message components natively.
+
```
Comment (almostSouji) [inline tags/tags.toml:717]: don't think we need that empty line? native bullet lists should have enough spacing

File: tags/tags.toml [hunk 5]
```diff
@@ -888,19 +890,21 @@ client.guilds.cache.reduce((acc, guild) => acc + guild.memberCount, 0)
...
     "auth-link",
 ]
 content = """
-**"Add to Server" button**
-- [Developer Portal](https://discord.com/developers/applications) > Your app > OAuth2 (General) > Default Authorization Link
-- Set authorization method to "In-app Authorization"
-- Check scopes `bot` and `applications.commands`
+**"Add App" button**
+- [Developer Portal](https://discord.com/developers/applications) > Your app > Installation
+- Choose "User Install" or "Guild Install"
```
Comment (almostSouji) [inline tags/tags.toml:903]: not exclusive

File: tags/tags.toml [hunk 6]
```diff
@@ -888,19 +890,21 @@ client.guilds.cache.reduce((acc, guild) => acc + guild.memberCount, 0)
...
+- Choose "Discord Provided Link" under "Install Link"
```
Comment (almostSouji) [inline tags/tags.toml:904]: turn around, location -> label is easier to comprehend

File: tags/tags.toml [hunk 7]
```diff
@@ -888,19 +890,21 @@ client.guilds.cache.reduce((acc, guild) => acc + guild.memberCount, 0)
...
+- Check scopes `bot` and choose permissions that your app *needs* under "Default Install Settings"
```
Comment (almostSouji) [inline tags/tags.toml:905]:
```suggestion
- Check the `bot` scope and choose any permissions that your app *needs* under "Default Install Settings"
```

**Review by almostSouji — CHANGES_REQUESTED** [review]

File: tags/tags.toml (hunk 4 above)
Comment (joeyk710) [reply tags/tags.toml:717]:
```suggestion

```

File: tags/tags.toml (hunk 6 above)
Comment (joeyk710) [reply tags/tags.toml:904]:
```suggestion
- "Install Link" > "Discord Provided Link"
```

**Pull request merged**